nilearn/interfaces/bids/query.py

```diff
@@ -206,10 +206,15 @@ def get_bids_files(
         inside the sub-label/ses-label directories.
 
     filters : :obj:`list` of :obj:`tuple` (:obj:`str`, :obj:`str`), optional
-        Filters are of the form (field, label). Only one filter per field
-        allowed. A file that does not match a filter will be discarded.
-        Filter examples would be ('ses', '01'), ('dir', 'ap') and
-        ('task', 'localizer').
+        Filters are of the form (field, label).
+        Only one filter per field allowed.
+        A file that does not match a filter will be discarded.
+        Filter examples would be
+        ``('ses', '01')``,
+        ``('dir', 'ap')`` or
+        ``('task', 'localizer')``.
+        If an empty string is passed ``('space', '')``,
+        then only files that do NOT contain that entity are returned.
```

Comment:
Can you give an example use case (here or in the doc itself) for how to use this filtering other than for the `space` entity ? I'm struggling to think of when I'd use this, but that's likely just my lack of imagination !

Reply:
It could apply to any entities in the raw dataset too: 

So pick any from one of those filename templates 
https://bids-website.readthedocs.io/en/latest/getting_started/folders_and_files/files.html#func

and imagine any data horror story you wish:

Data was reconstructed using different algorithm on the scanner side but the 'default' one was not 'flagged' with a `rec` entity in the filename (because "that's obvious that no rec entity means it is the base case!" said the person acquiring the data).

Same scenario when using contrast agent (`ce`), different acquisition parameters (`acq`), direction (`dir`)...

SO now if you want to select your "base case" you need to have a way to say: get me the files without that entity...

I agree that all these may still seem like edge cases but I have seen too many edge cases in the world of BIDS...

Reply:
Thanks for clarifying ! I can definitely see these edge-case scenarios, but I'm struggling to imagine a common use case and (as always my avoid-code-creep-at-all-cost conservatism is coming out when I say) I'm hesitant to continue expanding our BIDS functionality with the goal of covering edge-cases. 

I'm pro a short term fix for the examples, if needed, but I don't want to push that to user facing functionality (which would then have to go through deprecation cycles) if we think it's really only for edge-cases / until we fix the example datasets !

Reply:
OK I will keep the change but remove any advertisement of it

Reply:
Perfect, thank you !